src/styles.css

```diff
@@ -8,6 +8,102 @@ h1 {
   font-size: 55px;
 }
 
+/* Tag box */
+.tag-area {
+  background-color: #2c2d30;
+  height: 30vh;
+  width: 78%;
+  border-radius: 15px;
+  display: flex;
+  align-items: center;
+  overflow-y: hidden;
+  overflow-x: hidden;
+}
+
+.tag-rows {
+  height: 100%;
+  width: 100%;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.tag-row {
+  height: 40%;
+  width: 100%;
+  border-radius: 15px;
+  /* border: solid 2px white; */
```

Comment:
this commented out style still need to be here?

Reply:
Deleted